src/client/client/game.js

```diff
@@ -39,7 +39,10 @@ let game = function() {
 		_requestComplete,
 		_requestStart = Date.now();
 
-	let _toggleMarbleTracking = function(marble) {
+	let _toggleMarbleTracking = function(marble, forceTracking) {
```

Comment:
It's a hindsight, but since this is not a toggle function (like an on/off switch), I think `_trackMarble` makes more sense for the name.